pokemongo_bot/cell_workers/update_live_stats.py

```diff
@@ -344,6 +343,7 @@ def _get_stats_line(self, player_stats):
             ),
             'pokeballs_thrown': 'Threw {:,} pokeballs'.format(player_stats['pokeballs_thrown']),
             'stardust_earned': 'Earned {:,} Stardust'.format(player_stats['stardust_earned']),
+            'stardust_amount': 'Have {:,} Stardust'.format(self.bot.metrics.dust['latest']),
```

Comment:
yeah metrics.dust['latest'] appears to just call bot.stardust which is a bit dodgy